pos_return_order/README.rst

```diff
@@ -0,0 +1,101 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+PoS - Return Order
+==================
+
+This module was written to extend the functionality of odoo Point Of Sale about
+ Order returns.
+
+With this module, it is now forbidden to return more products than the quantity
+of the initial Order.
+
+A link is created between the returned Order and the initial Order.
+A link is created between the returned Order Line and the initial Order Line.
+
+Implemented Features
+--------------------
+
+* A wizard that allow to select just some products to return:
+
+.. image:: /pos_return_order/static/description/partial_return_wizard.png
+
+Implemented Constraints
+-----------------------
+
+* User can not return more products than the initial quantity:
+
+.. image:: /pos_return_order/static/description/returned_qty_over_initial.png
+
+* If a line has been partially refund, only a reduced quantity can be returned:
+
+.. image:: /pos_return_order/static/description/sum_returned_qty_over_initial.png
+
+* It is not possible to set a negative quantity if the initial Pos Order is
+  not indicated:
+
+.. image:: /pos_return_order/static/description/initial_pos_order_required.png
+
+Installation
+============
+
+Normal installation.
+
+Configuration
+=============
+
+In some case, you can want to let the possibility to allow negative quantity
```

Comment:
remove "can"
the possibility for negative quantities in a PoS order, regardless of the initial order.